src/main/java/com/algorand/sdkutils/listeners/TemplateGenerator.java

```diff
@@ -291,6 +291,9 @@ public void terminate() {
     public void onEvent(Publisher.Events event, TypeDef type) {
         switch(event) {
             case NEW_PROPERTY:
+                if (type.isOfType("enum")) {
```

Comment:
I'm not sure about this, could we change the go templates to skip over enums?

Ideally we would use the template approach for all SDKs, so at least the JavaSDK would need this event.

Reply:
Wouldn't the go SDK still need this, but use "string"?

Reply:
> Ideally we would use the template approach for all SDKs, so at least the JavaSDK would need this event.

I see your point...I'll take a look and see if it's easy to configure enum-specific behavior in the templates.

From what I can tell both the JS and Go Sdks just use string fields for enums so there is no need to take an action in addition to those we're already taking when generating paths, models, etc. like we are in Java.